infra/modules/service/main.tf

```diff
@@ -51,8 +51,8 @@ resource "aws_ecs_service" "app" {
   network_configuration {
     # TODO(https://github.com/navapbc/template-infra/issues/152) set assign_public_ip = false after using private subnets
```

Comment:
nit: This comment should probably be deleted with this change.

Reply:
got it thanks